src/BasicRum/Release.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum;
+
+use DateTime;
+
+class Release
+{
+    private $releasses;
+
+    public function __construct(\App\Repository\ReleasesRepository $releases)
+    {
+        $this->releases = $releases;
+    }
+
+    /**
+     * @param string $date1 Start date
+     * @param string $date2 End date
+     *
+     * @return array [description]
+     */
+    public function getAllReleasesBetweenDates(string $date1, string $date2): array
+    {
+        $dateArray['startDate'] = new  DateTime($date1);
```

Comment:
It doesn't make sense to use $dateArray here. You can just pass 2 arguments to findAllBetweenDates(new  DateTime($date1),  DateTime($date2)). Creating an array doesn't bring much value but increases the complexity a bit.